HARMONIZED_TEST_ENVIRONMENT.md

```diff
@@ -0,0 +1,72 @@
+Harmonized X-road test environment
+===================
+
+- LXD based common test development environment to Finland and Estonia
+	- Includes servers: CA, CENTRAL SERVER, MANAGEMENT SECURITY SERVER, SECURITY SERVERS
+	    - https://github.com/ria-ee/X-Road/tree/develop/ansible
+- Can be used for testing:
+	- UI and integrations, SOAP testing, service testing and Service mocks
+	- Not for performance testing!
+- Benefits:
+	- Tests are compatible between Estonia and Finland common test environments
+		- Re-usable tests
+	- Test can be easily imported to own specific environment from LXD
+	- Jenkins jobs are generated and we have harmonized test data in different test environments
```

Comment:
I didn't quite understand what means "Jenkins jobs are generated'? I guess that it should understand in a way that 'Jenkins jobs are generated (created) by the testing person who originally created the test case and by syncing the Jenkins jobs between countries via common Github repo, we have harmonized test data in different test environments'. Anyway, this sentence/line should be clarified so that the meaning is clear.

Reply:
changed